src/components/productListing/ProductsFilters.css

```diff
@@ -0,0 +1,62 @@
+@import url("https://fonts.googleapis.com/css2?family=Comfortaa:wght@700&family=Quicksand:wght@300&family=Roboto:wght@100;300;400;500&display=swap");
```

Comment:
If this font is being used through out the application, can be moved to index.css

Reply:
yes , you are right , but this font style is specifically used here only.